client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +137,22 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function getWpcomPlanTotalIfPaidMonthly(
```

Comment:
Could this be a hook? in which case it could call other hooks, no need to pass plans through.

Reply:
Done.